domain/src/main/java/com/example/domain/usecase/GetHeartRateUseCase.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.domain.usecase
+
+import com.example.domain.base.BaseUseCase
+import com.example.domain.model.HeartRateInfo
+import com.example.domain.model.Result
+
+interface GetHeartRateUseCase : BaseUseCase<String, HeartRateInfo> {
+    override suspend operator fun invoke(param: String): Result<HeartRateInfo>
+}
```

Comment:
Let's not use operators here, we might add some other function here

Reply:
Resolved